src/Umbraco.Web.UI.Client/src/less/tree.less

```diff
@@ -436,6 +436,9 @@ div.locked:before{
 }
 .umb-actions-child i {
 	font-size: 32px;
+    min-width: 32px;
+    text-align: center;
+    line-height: 24px; /* set line-height to ensure all icons use same line-height */
 }

```

Comment:
This ensure custom icons (thumbnails) in create dialog also is styled in same way as core icons. Increased the font-size a bit and decreased the margin. The "album" icon is a custom icon. Furthermore it also set the line-height, 32px would be 100% line-height, but it works great too with 24px and shrinks it a bit.

**Before**
![image](https://cloud.githubusercontent.com/assets/2919859/14226226/f5d6dd26-f8db-11e5-9015-9b0b8c35cde8.png)

**After**
![image](https://cloud.githubusercontent.com/assets/2919859/14226229/0ae2a43e-f8dc-11e5-9d61-73de3e2db138.png)